EDA.qmd

```diff
@@ -916,13 +925,13 @@ Typically, the first one or two arguments to a function are so important that yo
 The first two arguments to `ggplot()` are `data` and `mapping`, and the first two arguments to `aes()` are `x` and `y`.
 In the remainder of the book, we won't supply those names.
 That saves typing, and, by reducing the amount of boilerplate, makes it easier to see what's different between plots.
-That's a really important programming concern that we'll come back to in Chapter \@ref(functions).
+That's a really important programming concern that we'll come back to in [Chapter -@sec-functions].
```

Comment:
Why doesn't `@sec-functions` work here?

Reply:
This is how Chapter referencing works in Quarto book, it's a slightly awkward syntax I think. See bottom of the section https://quarto.org/docs/books/book-crossrefs.html#creating-references for chapters.